experimental/packages/exporter-trace-otlp-grpc/src/util.ts

```diff
@@ -131,6 +134,99 @@ export function validateAndNormalizeUrl(url: string): string {
   return target.host;
 }
 
+export function configureSecurity(credentials: grpc.ChannelCredentials | undefined, endpoint: string):
+  grpc.ChannelCredentials {
+
+  let insecure: boolean;
+
+  if (credentials) {
+    return credentials;
+  } else if (endpoint.startsWith('https://') || endpoint.startsWith('grpcs://')) {
```

Comment:
I thought we had agreed not to use grpc/grpcs scheme?

Reply:
Oh. I think I misunderstood your reply about the grpc(s) scheme:

"Spec states we should accept whatever the underlying implementation accepts. If we are passed a string, we should pass it directly to the underlying implementation. If it fails then fine. If it works then fine."

So basically you're saying that we should let a user pass whatever string they want with any scheme but we shouldn't check for grpc(s) scheme since we only care about http(s). Is that correct?

Reply:
yes I think so

Reply:
Done